modules/virt-importing-vm-datavolume.adoc

```diff
@@ -93,21 +94,22 @@ spec:
               bus: virtio
             name: datavolumedisk1
         machine:
-          type: "" <4>
+          type: "" <5>
         resources:
           requests:
             memory: 64M
-      terminationGracePeriodSeconds: 0
+      terminationGracePeriodSeconds: 60
       volumes:
       - dataVolume:
           name: fedora-dv
         name: datavolumedisk1
 status: {}
 ----
-<1> The `HTTP` source of the image you want to import.
-<2> The `secretRef` parameter is optional.
-<3> The `certConfigMap` is required for communicating with servers that use self-signed certificates or certificates not signed by the system CA bundle. The referenced ConfigMap must be in the same namespace as the DataVolume.
-<4> Specify `type: dataVolume` or `type: ""`. If you specify any other value for `type`, such as `persistentVolumeClaim`, a warning is displayed, and the virtual machine does not start.
+<1> The source type from which to import the image. This example uses a HTTP endpoint. To import a container disk from a registry, replace `http` with `registry`.
+<2> The source of the virtual machine image you want to import. This example references a virtual machine image at an HTTP endpoint. An example of a container registry endpoint is `url: "docker://kubevirt/fedora-cloud-container-disk-demo:latest"`.
+<3> The `secretRef` parameter is optional.
+<4> The `certConfigMap` is required for communicating with servers that use self-signed certificates or certificates not signed by the system CA bundle. The referenced ConfigMap must be in the same namespace as the DataVolume.
+<5> Specify `type: dataVolume` or `type: ""`. If you specify any other value for `type`, such as `persistentVolumeClaim`, a warning is displayed, and the virtual machine does not start.
 
```

Comment:
There doesn't seem to be anything documenting the insecure-registries config map as written here: https://github.com/kubevirt/containerized-data-importer/blob/master/doc/image-from-registry.md#insecure-registry

Reply:
That is quite the oversight on my behalf. Lemme get right onto that.

Reply:
@aglitke I've added the procedure into the same assembly as creating the container disks:
https://cnv-1403-condensed--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.html#virt-disabling-tls-for-registry_virt-using-container-disks-with-vms
Does that make sense to you?

There's an added prerequisite in this assembly as well for visibility:
`If the container registry does not have TLS security you must xref:../../../virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.adoc#virt-disabling-tls-for-registry_virt-using-container-disks-with-vms[add the registry to the `cdi-insecure-registries` ConfigMap] before you can import a container disk from it.`